scrapinghub/client.py

```diff
@@ -630,6 +632,8 @@ def update_tags(self, add=None, remove=None):
 
             >>> job.update_tags(add=['consumed'])
         """
+        if not (add or remove):
+            raise ValueError('Please provide tags to add or remove')
```

Comment:
yeah, redundancy is my main concern. when data is validated server side - you only need to change it in one place later